network-api/networkapi/management/commands/heroku_release.py

```diff
@@ -1,16 +1,16 @@
-from django.core.management.base import BaseCommand

```

Comment:
I think this is because I did a global CR+LF => LF line ending replacement... were CR+LF crucial here?

Reply:
probably not